x-pack/plugins/uptime/public/lib/alert_types/translations.ts

```diff
@@ -49,3 +49,23 @@ Aging Certificates: {agingCommonNameAndDate}
     defaultMessage: 'Uptime TLS',
   }),
 };
+
+export const DurationAnomalyTranslations = {
+  defaultActionMessage: i18n.translate('xpack.uptime.alerts.durationAnomaly.defaultActionMessage', {
+    defaultMessage: `A {severity} level anomaly with score {severityScore} was detected at {anomalyStartTimestamp} on {monitor} response duration in Uptime at url {monitorUrl}.
+Response times as high as {slowestAnomalyResponse} have been detected from location {observerLocation}. Expected response time is {expectedResponseTime}.`,
+    values: {
+      severity: '{{state.severity}}',
+      anomalyStartTimestamp: '{{state.anomalyStartTimestamp}}',
+      monitor: '{{state.monitor}}',
+      monitorUrl: '{{{state.monitorUrl}}}',
```

Comment:
I've noticed in the server log output the URL is escaped like so: `https:&#x2F;&#x2F;www.android.com ` is this intentional?

Reply:
for me it works fine in slack. May be some issue with server logs.
![image](https://user-images.githubusercontent.com/3505601/87330896-83025f80-c539-11ea-8e6c-410d36d64fba.png)

Reply:
This is an alerting level issue, not this PR's concern.